servant-minimalist.hsfiles

```diff
@@ -0,0 +1,232 @@
+{-# START_FILE package.yaml #-}
```

Comment:
I don't want `package.yaml`. it divides community.

Please change to `{{name}}.cabal`.